src/server/gdb_server.c

```diff
@@ -390,7 +390,7 @@ static int gdb_put_packet_inner(struct connection *connection,
 			break;
 		}
 
-		LOG_WARNING("Discard unexpected char %c", reply);
+		LOG_DEBUG("Discard unexpected char %c", reply);
```

Comment:
Why did you make this change? Since it's in the target-independent part of the code, we'll have to give some justification when I merge this to the official OpenOCD branch.

Reply:
This message is noise for the user and is of debug character and not warning, as the user has no possibilities of doing anything about it.

Reply:
I will try to push this upstream, but the counter argument is probably that this message indicates a bug either in OpenOCD or the debugger that's connected to it.

Reply:
FYI, in my latest release of GME OpenOCD I'm not carrying in any of the changes you make in the target-independent code until they are upstreamed.